dependabot_time_to_merge.rb

```diff
@@ -0,0 +1,90 @@
+raise "You need to provide two dates in the following format: 2022-06-23" if ARGV.length != 2
+
+require "octokit"
+require "net/http"
+require "json"
+require "date"
+
+def client
+  @client ||=
+    Octokit::Client.new(
+      access_token: ENV.fetch("GITHUB_TOKEN"),
+      auto_paginate: false
+    )
+end
+
+def govuk_repos
+  @govuk_repos ||= 
+    JSON.parse(
+      Net::HTTP.get(
+        URI("https://docs.publishing.service.gov.uk/repos.json")
+      )
+    )
+      .map { |repo| "alphagov/#{repo["app_name"]}" }
+end
+
+def dependabot_prs
+  @count = 0
+  client.search_issues("is:pr user:alphagov is:closed author:app/dependabot archived:false", per_page: 100)
+  
+  last_response = client.last_response
+
+  return [] if last_response.data.items.empty?
+
+  pulls = []
+  pulls << last_response.data.items
+  puts "Fetching first page of results... (API call ##{@count += 1})"
+
+  until last_response.rels[:next].nil?
+    sleep 60 if (current_page(last_response) % 3).zero?
+    last_response = last_response.rels[:next].get
+    pulls << last_response.data.items
+    puts "Fetching next page of results... (#{api_calls_counter})"
+  end
+  pulls.flatten
+end
+
+def api_calls_counter
+  @count += 1
```

Comment:
We can handle both the instantiation and the increment by doing something like:

```suggestion
  @count = defined?(@count) ? 1 : (@count + 1)
```

Reply:
Oh nice, I like that!

Reply:
It would work with `@count = @count ? @count + 1 : 1`, but I can actually just use the current_page method and subtract 1 from it